lib/ui/text/paragraph_builder.h

```diff
@@ -83,6 +83,8 @@ class ParagraphBuilder : public RefCountedDartWrappable<ParagraphBuilder> {
                             const std::u16string& ellipsis,
                             const std::string& locale);
 
+  txt::ParagraphStyle style_;
```

Comment:
Is there an advantage to making the `style` and `font_collection` members of `ParagraphBuilder`?

AFAICT they are not reused after the `ParagraphBuilder` constructor returns.

Reply:
No, this is from a different experiment where I was re-creating the builder because reset wasn't available. I'll revert this part of the patch.